server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -1100,12 +1122,35 @@ export function configureWebSocketServices(
 							: [contentBatch];
 
 						for (const content of contents) {
-							const signalMessage: ISignalMessage = {
-								clientId,
-								content,
-							};
+							let signalMessage: ISignalMessage;
+							let roomId: string;
+							const feature_submit_signals_v2 = "submit_signals_v2";
```

Comment:
Typically put global-ish consts like this outside of the function, at the top of the file (the extra memory usage is negligible in this case, but it's a nice norm to follow).